frontend/src/features/admin-form/responses/ChartsPage/UnlockedCharts/UnlockedChartsContainer.tsx

```diff
@@ -195,6 +195,30 @@ export const UnlockedChartsContainer = () => {
         <VStack divider={<Divider />} gap="1.5rem">
           {renderedCharts}
         </VStack>
+      ) : filteredDecryptedData.length === 0 ? (
+        <Container p={0} maxW="42.5rem">
+          <Stack spacing="1rem" align="center">
+            <Text
+              as="h2"
+              color="primary.500"
+              textStyle="h2"
+              whiteSpace="pre-wrap"
+            >
+              No charts generated for this date range
+            </Text>
+            <Text
+              textStyle="body-1"
+              color="secondary.500"
+              mb="0.5rem"
+              align="center"
+            >
+              There may not have been any responses collected within this date
```

Comment:
question: why may?

suggestion:  There were no responses collected within this date range.

Reply:
Great catch. Updated in fe93acfd4